db/migrate/20180614143214_add_collection_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddCollectionToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :collection, :string, array: true, default: "{}"
```

Comment:
I would suggest creating a new class called `Collection` and add a reference to the User class. Should a user have one Collection or many? .